website/docs/events/brainstorming-2023-06-22.md

```diff
@@ -0,0 +1,62 @@
+# Brainstorming: Education in the Digital Age
+
+Good education is always a challenge, even more so in the digital age, where information is readily available and the potential of automation is disrupting existing methods.
+Questions come about:
+
+* What are good digital education materials?
+* Which materials should be used live (during a class, lecture) or which should be used offline?
+* What should happen face-to-face and what should happen online?
+* What are actions that are important to automate?
+* How do we structure content in repositories?
+* How is content available to trainees?
+
+And many more.
+
+At Open Education Hub, we think about these items and we aim to create the future of education, given the challenges and opportunities of the digital age, and with the support of all those involved.
+
+On **Thursday, June 22, 2023, starting with 15:00 EEST**, we organize a brainstorming meeting to collect ideas and viewpoints from educators regarding education in the digital age.
+Our goal is to extract best practices, guides and lessons learned in order to build up the methods, approaches and infrastructure for the future of education, a future that will be tied to digitalization and technology.
+
+In this 2.5 hours **in-person** discussion, we welcome anyone interested in sharing their ideas and in improving their (and others') digital educational contents, infrastructure and teaching methods.
+We will have drinks and sweets.
+
+We will spend about one hour discussing the usefulness of slides in the digital age.
+Then, for the next part, we will split in breakout groups to discuss several topics;
+and, at the end, present those topics to the others.
+We will draw conclusions and then announce them to the greater education community.
+
+If you plan to attend, please fill [this form](https://forms.gle/kG3ioVqMJR8526sB9).
+
+## Schedule
+
+* Date: Thursday, June 22, 2023
+* Time: 15:00-17:30 EEST (Romania time)
+* Place: Room PR105, [PRECIS Center](https://goo.gl/maps/cBXziUixXDXNfxko6), Faculty of Automatic Control and Computer Science, University POLITEHNICA of Bucharest
+
+## Agenda
+
+* 15:00-15:10: Welcome
+* 15:10-16:00: Brainstorming: Are slides (still) useful in teaching and learning in the digital age?
+* 16:00-16:15: Break
+* 16:15-17:00: Breakout groups
+  1. Structuring a class digital content repository:
+     What should be the layout of digital contents for a class?
+     Where do slides, text, skeleton code, demos, images, go?
+  1. Public vs private content:
+     What content should be available to trainees, what should remain private (such as exam questions)?
```

Comment:
Maybe detail a bit on the fact that we can have lets say part of the exam questions public, it can all be between nothing public and all public.
It feels like the question now is a bit strict, it would get answers in the form of `Private stuff: exam questions, homework solutions, public stuff: lecture items, lab items`, nothing in-between.